files/puppet_helper.groovy

```diff
@@ -170,10 +170,13 @@ class Actions {
       )[0].getStore()
 
     def credentials
+    if (id == "") {
+      id = null
+    }
```

Comment:
Actually, that might not make any sense.  I'm not sure how you would pass null from the cli.